src/Panel/Ui/Buttons/Buttons.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Kirby\Panel\Ui\Buttons;
+
+use Kirby\Cms\App;
+use Kirby\Panel\Model;
+use Kirby\Toolkit\A;
+
+/**
+ * View buttons
+ * @since 5.0.0
+ *
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Buttons
+{
+	public function __construct(
+		protected readonly string $view,
+		protected array|null $buttons = null
+	) {
+		$this->buttons ??= App::instance()->option(
+			'panel.viewButtons.' . $view
+		);
+	}
+
+	/**
+	 * Sets the default buttons
+	 *
+	 * @return $this
+	 */
+	public function defaults(string ...$defaults): static
+	{
+		$this->buttons ??= $defaults;
+		return $this;
+	}
```

Comment:
`Question`: What do you think about this syntax to pass defaults to the object? Allowing this:

```php
Buttons::view('test')->defaults('a', 'b', 'c')->render();
```

I think it makes the syntax attractive, but one might see it as a bit uncommon for Kirby.